pom.xml

```diff
@@ -33,10 +45,29 @@
       <dependency>
         <groupId>org.jboss.arquillian</groupId>
         <artifactId>arquillian-bom</artifactId>
-        <version>1.7.0.Final</version>
+        <version>${arquillian-bom.version}</version>
         <scope>import</scope>
         <type>pom</type>
       </dependency>
+      <dependency>
```

Comment:
These were moved around for no reason. Please put back how it was. Parameterizing the version is fine.